app/javascript/packs/managers/cuisines.js

```diff
@@ -0,0 +1,85 @@
+$(document).ready(function () {
+  let action_name = $('body').data('action')
+  console.log(action_name)
+});
+
```

Comment:
使ってるファイルでimportしてください。

```suggestion
import Sortable from "sortablejs"
```